test/test_sparse.py

```diff
@@ -4458,6 +4458,17 @@ def test_sum_meta(self, dtype, layout):
             self.assertTrue(r.is_meta)
             self.assertEqual(r.shape, ())
 
+    @all_sparse_layouts('layout', include_strided=False)
+    @parametrize("dtype", [torch.float64])
+    def test_add_meta(self, dtype, layout):
+        device = 'cpu'
+        index_dtype = torch.int64
+        for t in self.generate_simple_inputs(layout, device=device, dtype=dtype, index_dtype=index_dtype):
+            m = t.to(device='meta')
+            r = torch.add(m, m)
+            self.assertTrue(r.is_meta)
```

Comment:
```suggestion
            self.assertEqual(r, m)
```
that will also check the shape and dtype properties of the tensors.

Reply:
good idea, done!